tools/generator/src/Generator/AddTargets.swift

```diff
@@ -169,6 +174,32 @@ Product for target "\(key)" not found in `products`
         return script
     }
 
+    private static func createLinkParamsScript(
+        in pbxProj: PBXProj,
+        linkParamsFile: String?
+    ) throws -> PBXShellScriptBuildPhase? {
+        guard let linkParamsFile = linkParamsFile else {
+            return nil
+        }
+
+        let script = PBXShellScriptBuildPhase(
+            name: "Create link.params",
+            inputPaths: [linkParamsFile],
+            outputPaths: ["$(DERIVED_FILE_DIR)/link.params"],
+            shellScript: #"""
+sed \
+  -e "s|^\(.*\)\$(BUILD_DIR)\(.*\)\$|\"\1${BUILD_DIR}\2\"|g" \
+  -e "s|^\(.*\)\$(DEVELOPER_DIR)\(.*\)\$|\"\1${DEVELOPER_DIR}\2\"|g" \
+  "$SCRIPT_INPUT_FILE_0" > "$SCRIPT_OUTPUT_FILE_0"
+
+"""#,
```

Comment:
Build env variables aren't available once clang has this file, so we need to process the link.params first before handing it off to clang.